travel_platform/travel_app/views.py

```diff
@@ -1,17 +1,19 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import json
 import requests
 import sys
 
-from django.http import JsonResponse
+from config_parser.config_reader import get_setting
+from django.http import JsonResponse, HttpResponse
 from django.conf import settings
 from django.shortcuts import render
 from django.views.decorators.cache import cache_page
 
 from api.izi_travel.izi_travel_data import *
-from config.conf import CONF_PATH
-
+from config.conf_open_maps import CONF_PATH_OM
+from api.foursquare.foursquare_data import *
 
 # FIXME RomanPryima: remade sys path in a properly way
```

Comment:
Try it like this:
import os
sys.path.append(os.path.dirname((
                    os.path.dirname(
                        os.path.dirname(
                            os.path.abspath(__file__))))))    # for config_parser package

sys.path.append(os.path.dirname
                  (os.path.dirname(
                       os.path.abspath(__file__))))    # for izi_travel package